src/java/org/streamingpool/core/support/AbstractStreamSupport.java

```diff
@@ -71,4 +74,8 @@ public ProvidingService providingService() {
         return providingService;
     }
 
+    @Override
+    public InstrumentationService dependencyService() {
```

Comment:
consider renaming to instrumentationService

Reply:
... On the other hand ... Why is this exposed here? since this is some support class, one could also consider simple exposing the poolContent() method or similar...

Reply:
renamed :+1:  if we remove it from the pool content then I would not expose it :)